### PR TITLE
DR-1483: Move initial delay seconds to variable, set default

### PR DESCRIPTION
**Pull request by snf2ye**

### CODE REVIEW

**Review by smark88 — APPROVED** [review]

**Review by nmalfroy — APPROVED** [review]
lgtm!

**Comment by snf2ye** [comment]
Action failure documented in DR-1519

**Review by ddietterich — APPROVED** [review]
lgtm - thank you!

File: charts/datarepo-api/Chart.yaml [hunk 1]
```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.0.22
-appVersion: 0.0.22
+version: 0.0.23
+appVersion: 0.0.23
 
 # This is the version number of the application being deployed. This version number should be
```
Comment (pshapiro4broad) [inline charts/datarepo-api/Chart.yaml:20]: This comment seems bogus.

**Review by pshapiro4broad — APPROVED** [review]

**Comment by snf2ye** [comment]
Update confirmed manually w/ my personal dev deployment.

**Pull request merged**